bson/primitive/objectid_test.go

```diff
@@ -28,6 +28,13 @@ func TestString(t *testing.T) {
 	require.Contains(t, id.String(), id.Hex())
 }
 
+func BenchmarkHex(b *testing.B) {
```

Comment:
Hello @isopov, thanks so much for your contribution! It's a reasonable PR. I saw an improvement in my environment as well. How do you like to put the benchmark here? I will merge your code when the build passes.

Reply:
Sure - adding benchmark is fine. Though I think the only usage of it may be to catch the moment when go compiler will be complex enough to optimize the previous simpler method body and this hack may be reverted.

Reply:
Right, and it also helps to imply the season of this less straightforward solution and indicate accidental de-optimization.